File: MantikBaglaclari.php
<html>
<form action="" method = "post" />
<input type="text" name="sayi1">
<input type="text" name="sayi2">
<input type="text" name="sayi3">
<button>Gönder</button>
</form>
</html>

<?php 

if($_POST){




	$sayi1 = @$_POST["sayi1"];
	$sayi2 = @$_POST["sayi2"];
	$sayi3 = @$_POST["sayi3"];

if(!empty($sayi1 && $sayi2 && $sayi3)){

 if ($sayi1 > $sayi2 && $sayi1 < $sayi3) {
            echo $sayi1." sayısı 2. sayı ve 3. sayının arasındadır.<br />";
        } else {
            echo $sayi1." sayısı 2. sayı ve 3. sayının arasında değildir.<br />";
        }
        //İlk Girilen Sayı 2. Sayı ile eşit ve 3. Sayıdan Küçük mü?
        if ($sayi1 == $sayi2 && $sayi1 < $sayi3) {
            echo $sayi1." sayısı 2. sayı ile eşit ve 3. sayıdan küçüktür.<br />";
        } else {
            echo $sayi1." sayısı 2. sayı ile eşit ve 3. sayıdan küçük değildir.<br />";
        }
        //İlk Girilen Sayı 2. Sayı veya 3. Sayıdan Büyük mü?
        if ($sayi1 > $sayi1 || $sayi1 > $sayi3) {
            $sayi1." sayısı 2. sayı veya 3. sayıdan büyüktür.<br />";
        } else {
            echo $sayi1." sayısı 2. sayı veya 3. sayıdan büyük değildir.<br />";
        }
        //Girilen Sayılar Birbirlerine Eşit mi?
        if ($sayi1 == $sayi2 && $sayi2 == $sayi3) {
            echo "Girilen sayılar birbirlerine eşit.<br />";
        } else {
            echo "Girilen sayılar birbirlerine eşit değildir.<br />";
        }

    


}else{


echo "tüm alanlar dolu olmalıdır.";


}

}

   ?>

File: MantikBaglaclari.java
package mantikbaglaclari;

import java.util.Scanner;

public class MantikBaglaclari {

    public static void main(String[] args) {
        Scanner a = new Scanner(System.in);
        int sayi1, sayi2, sayi3;
        System.out.print("İlk Sayıyı Giriniz: ");
        sayi1 = a.nextInt();
        System.out.print("2. Sayıyı Giriniz: ");
        sayi2 = a.nextInt();
        System.out.print("Son Sayıyı Giriniz: ");
        sayi3 = a.nextInt();

        //İlk Girilen Sayı 2. Sayı ve 3. Sayı değerinin arasında mı?
        if (sayi1 > sayi2 && sayi1 < sayi3) {
            System.out.println(sayi1 + " sayısı 2. sayı ve 3. sayının arasındadır.");
        } else {
            System.out.println(sayi1 + " sayısı 2. sayı ve 3. sayının arasında değildir.");
        }
        //İlk Girilen Sayı 2. Sayı ile eşit ve 3. Sayıdan Küçük mü?
        if (sayi1 == sayi2 && sayi1 < sayi3) {
            System.out.println(sayi1 + " sayısı 2. sayı ile eşit ve 3. sayıdan küçüktür.");
        } else {
            System.out.println(sayi1 + " sayısı 2. sayı ile eşit ve 3. sayıdan küçük değildir.");
        }
        //İlk Girilen Sayı 2. Sayı veya 3. Sayıdan Büyük mü?
        if (sayi1 > sayi1 || sayi1 > sayi3) {
            System.out.println(sayi1 + " sayısı 2. sayı veya 3. sayıdan büyüktür.");
        } else {
            System.out.println(sayi1 + " sayısı 2. sayı veya 3. sayıdan büyük değildir.");
        }
        //Girilen Sayılar Birbirlerine Eşit mi?
        if (sayi1 == sayi2 && sayi2 == sayi3) {
            System.out.println("Girilen sayılar birbirlerine eşit.");
        } else {
            System.out.println("Girilen sayılar birbirlerine eşit değildir.");
        }
    }
}
